runtime/integration-tests/src/generic/env.rs

```diff
@@ -61,7 +61,11 @@ impl<Event> Blocks<Event> {
 /// Define an environment behavior
 pub trait Env<T: Runtime> {
 	/// Load the environment from a storage
-	fn from_storage(parachain_storage: Storage, sibling_storage: Storage) -> Self;
+	fn from_storage(
+		relay_storage: Storage,
+		parachain_storage: Storage,
+		sibling_storage: Storage,
+	) -> Self;
```

Comment:
Because in most tests, specifically in non-xcm tests, the typical usage will be initializing only the parachain storage, could we have two different environment initializations to simplify each test case? I mean:

```rust
// Just set the parachain storage
fn from_parachain_storage(storage: Storage) -> Self; 

// Set all storages
fn from_storage(
	relay_storage: Storage,
	parachain_storage: Storage,
	sibling_storage: Storage,
) -> Self;

Reply:
Yup, was also thinking to implement `Default` for this, since that will be used mainly from what I can tell.

Reply:
Great! 👌🏻